exporters/auto/src/main/java/com/google/cloud/opentelemetry/auto/GoogleCloudSpanExporterFactory.java

```diff
@@ -0,0 +1,25 @@
+package com.google.cloud.opentelemetry.auto;
+
+import com.google.cloud.opentelemetry.trace.TraceExporter;
+import io.opentelemetry.javaagent.tooling.TracerInstaller;
+import io.opentelemetry.javaagent.tooling.exporter.ExporterConfig;
+import io.opentelemetry.javaagent.tooling.exporter.SpanExporterFactory;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GoogleCloudSpanExporterFactory implements SpanExporterFactory {
+
+    private static final Logger log = LoggerFactory.getLogger(TracerInstaller.class);
```

Comment:
Why not `GoogleCloudSpanExporterFactory.class`?

Reply:
Ah, this is some copy paste from when I implemented this in the OpenTelemetry repo. Will update.